modules/server/src/test/scala/higherkindness/compendium/CompendiumArbitrary.scala

```diff
@@ -31,9 +33,19 @@ trait CompendiumArbitrary {
       .nonEmptyListOf(
         Gen.oneOf(Gen.alphaNumChar, Gen.const('-'), Gen.const('.'))
       )
+      .suchThat(_.length > 10)
```

Comment:
Just curious about this restriction.

Reply:
This is funny, tests were failing because one of the generated ids was `.` so it was interpreted as the current file, not a filename. I could solve it with a `_.lenght > 1`, but I considered that 10 were more elegant and real than a file called `.1` 😂